static/include/services/apis/sensors.md

```diff
@@ -0,0 +1,4 @@
+Method Name | Description
+----------- | -----------
+[`GetSensors`](/services/sensors/#getsensors) | Returns a list of names of the available sensors.
```

Comment:
```suggestion
[Sensors](/services/sensors/#sensors) | Returns a list of names of the available sensors.
```
Think not opt. -- See comment above about https://pkg.go.dev/go.viam.com/rdk/services/sensors#Service!

Reply:
See my comment on the other one--we use the proto definition, not the Go SDK one, for other APIs, so this should follow that too.